### PR TITLE
[student-libraries] Added library description to library display

### PR DESCRIPTION
**Pull request by jmkulwik**
# Description
Stores and displays any descriptions a user has added to a library.

LibraryDescription

## Links
- [spec](https://docs.google.com/document/d/1VxMmdHdIeBWCXKofkjEy9MuWcrzeEW3Zt-Ir8pINYdA/edit)
- [jira](https://codedotorg.atlassian.net/browse/STAR-710)
- [all relevant PRs](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3Ajmkulwik+archived%3Afalse+student+libraries)
## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/code-studio/initApp/project.js [hunk 1]
```diff
@@ -553,6 +564,19 @@ var projects = (module.exports = {
       this.setTitle(newName);
     }
   },
+  deleteLibraryData(callback) {
```
Comment (jmkulwik) [inline apps/src/code-studio/initApp/project.js]: Will be needed for when we implement "Unpublish"

Comment (ajpal) [reply apps/src/code-studio/initApp/project.js]: If this is unused right now, can you remove it from this PR and add it when you do unpublish?

**Review by ajpal — APPROVED** [review]

File: apps/src/code-studio/components/libraries/LibraryListItem.jsx [hunk 2]
```diff
@@ -89,7 +85,7 @@ export default class LibraryListItem extends React.Component {
           </button>
         )}
         <span style={styles.description}>
-          {this.displayDescription(library.description)}
+          {library.description}
           <br />
           {library.studentName && (
             <span style={styles.author}>Author: {library.studentName}</span>
```
Comment (joshlory) [inline apps/src/code-studio/components/libraries/LibraryListItem.jsx:91]: Nit, not related to this PR: localization of this string?

Comment (jmkulwik) [reply apps/src/code-studio/components/libraries/LibraryListItem.jsx:91]: Good call. I made a task go do a final pass of library UI elements and localize them.
https://codedotorg.atlassian.net/browse/STAR-788

File: apps/src/code-studio/initApp/project.js (hunk 1 above)
Comment (jmkulwik) [reply apps/src/code-studio/initApp/project.js]: 👍

**Pull request merged**